src/UseCase/BannerProvideUseCase.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\BannerServer\UseCase;
+
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * @license GNU GPL v2+
+ */
+class BannerProvideUseCase {
+	public function processBannerRequest( BannerProvideRequest $request ): Response {
```

Comment:
When our use cas is a sentence, I'd name the methods as verbs, e.g. `provide` or `select`. Or repeat the use case name as a method. But that's just me, we can discuss this and I'll do some reserach on the clean architecture.